server/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "../dist/",
+    "types": [],
+    "typeRoots": [
+      "node_modules/@types"
+    ],
+    "target": "es6",
+    "lib": [
+      "esnext.asynciterable",
+      "es6",
+      "es2015.promise",
+      "dom"
```

Comment:
You don't need to import the DOM library here unless you also have client-side JS code to generate. Otherwise it imports a bunch of random stuff that isn't actually available in a Node.js environment